Sources/ActionHolder.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public struct ActionHolder <A: Equatable> {
```

Comment:
Definitely makes sense to me. 
I think we should do it for all action generics across the project. 
The only place where I think brevity is more important, is a function's generic type, so I'd suggest to proceed use single characters for function generic names